packages/ui/src/kit/VAccordion/VAccordionItem.theme-default.scss

```diff
@@ -0,0 +1,46 @@
+.v-accordion-item.theme-default {
+  cursor: pointer;
+  border-top: 1px solid #dadde4;
+  transition: all 150ms ease-in-out;
+
+  &:last-child {
+    border-bottom: 1px solid #dadde4;
+  }
+
+  &:hover {
+    background-color: rgba(213, 217, 224, 0.25);
+  }
+
+  &:focus {
+    background-color: rgba(110, 50, 201, 0.15);
+    border-top: 1px solid #6e32c9;
```

Comment:
the [same](https://github.com/endpass/endpass-core/pull/302/files#diff-4bad877a5777710bd49b601023625a21R11)

Reply:
Rename colors into human-readable ones, and I will use it instantly.

Reply:
@ifiri we use --endpass-ui-color-grey-2 and we have reasons for that. Please, use it too